src/Cool/String/String.cpp

```diff
@@ -295,7 +388,13 @@ static auto value_from_string_impl_vec(std::string_view str)
 {
     std::vector<std::pair<size_t, size_t>> words_positions;
 
-    std::optional<size_t> ptr_in_string = 0; // Offset by 1 to skip the parenthesis
+    std::optional<size_t> ptr_in_string = 0;
+
+    auto opening_parenthesis_position = str.find_first_of("(");
+    if (opening_parenthesis_position != std::string_view::npos)
+    {
+        str = substring(str, opening_parenthesis_position, str.length());
```

Comment:
If you make ptr_in_string = opening_parenthesis_position you don't need to call substring